collect_app/src/main/res/values/strings.xml

```diff
@@ -664,4 +664,6 @@
     <string name="background_location_disabled">This form wants to track your location but tracking is disabled. Please enable in the \u0020\u0020⋮\u0020\u0020 menu above.</string>
     <string name="background_location_enabled">This form tracks your location. You can disable tracking in the \u0020\u0020⋮\u0020\u0020 menu above.</string>
     <string name="open_file">Open file</string>
+    <string name="sms_feature_disabled_dialog_title">SMS feature disabled</string>
+    <string name="sms_feature_disabled_dialog_message">Sending forms via SMS has been disabled in v1.20.0 due to the new Google\'s permission policy. We will try to bring the feature back soon.</string>
```

Comment:
I wonder if we can be more helpful here, e.g. `Sending forms via SMS has been temporarily disabled due to Google\'s new permission policy as of Collect version 1.20.0. We will try to bring the feature back soon. If you depend on this feature, you can downgrade to version 1.19.0`

Does ODK host downloadable APKs anywhere for users without access to the play store?

Reply:
We did a dialog for the removal of current() that I thought was great. You can find the PR https://github.com/opendatakit/collect/pull/2524/files.

In this case, the title should be "Sending via SMS Disabled" and the message should be "Sending submissions via SMS has been disabled due to Google Play Store restrictions. \n\nPlease make sure the person who configured this device reads the details below and takes action."

Then link to https://forum.opendatakit.org/t/17973 (the ID of a post that I'll write). This will allow us to put whatever actions people need to take there (including linking to APKs).